t/FHEM/00_SIGNALduino/02_SIGNALduino_calcRSSI.t

```diff
@@ -42,9 +42,10 @@ InternalTimer(time()+1, sub {
 		note($rmsg);
 		my %signal_parts=SIGNALduino_Split_Message($rmsg,$targetHash->{NAME});
 		my $rssi=$signal_parts{rssi};
+		my $rssiStr;
```

Comment:
diese Zeile müsste auch weg damit es zu keinem warning kommt oder?

Reply:
Die Zeile 45 `$rssiStr` (in der Änderung) wurde von mir gesetzt weil in der Zeile 44 die Variable `$rssi` schon gesetzt wird. Da ich mit der Schreibweise `my ($rssi,$rssiStr)` beide mit my versehe, aber `$rssi` davon schon gesetzt ist, so habe ich das my weggenommen und `$rssiStr` oben mit my gesetzt. ;-) In dem folgenen Codeausschnitt siehst du es deutlich.

https://github.com/RFD-FHEM/RFFHEM/blob/a07524aaeaa41a897af5d952a00927cbd1050cf4/t/FHEM/00_SIGNALduino/02_SIGNALduino_calcRSSI.t#L40-L50

Reply:
ach ja das my in Zeile 48 wird ja entfernt. Hatte wohl Tomaten auf den Augen ;-)